Content.Server/Chemistry/ReagentEffects/Polymorph.cs

```diff
@@ -0,0 +1,36 @@
+using Content.Server.Mind.Components;
+using Content.Server.Polymorph.Components;
+using Content.Server.Polymorph.Systems;
+using Content.Shared.Chemistry.Reagent;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Chemistry.ReagentEffects;
+
+public sealed partial class Polymorph : ReagentEffect
+{
+
+    /// <summary>
+    ///     What polymorph prototype is used on effect
+    /// </summary>
+    [DataField("prototype")] public string PolymorphPrototype { get; set; }
+
+    protected override string? ReagentEffectGuidebookText(IPrototypeManager prototype, IEntitySystemManager entSys)
+        => Loc.GetString("reagent-effect-guidebook-make-tree", ("chance", Probability));
```

Comment:
This shouldn't be hardcoded for trees description-wise.

Reply:
resolved